### DISCUSSION TITLE
Stuck at "Downloading LFS objects: 100%"; .git/lfs/tmp usage increasing

### DISCUSSION THREAD

**Discussion by kmn1024** [git-lfs/git-lfs#5859] [Q&A]
I'm using a command like the one below, to download only a portion of a repository:
```
GIT_LFS_SKIP_SMUDGE=1 git clone ...
git lfs pull --include="000[0-1]*.tar" --exclude=""
```

After "Downloading LFS objects" reaches 100%, I can see all the files fully downloaded under the main directory and the .git/lfs/objects directory. .git/lfs/incomplete stays empty.

The strange thing is, the command doesn't exit, and I see .git/lfs/tmp increasing in size. It looks like this:
```
>> ls -ltr
...
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1626005411
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1526488038
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1423887663
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1381271033
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1378017958
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1300972657
-rw-rw-rw- 1 root root          0 Sep  5 18:50 116858538
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1150194246
-rw-rw-rw- 1 root root          0 Sep  5 18:50 1105117604
-rw-rw-rw- 1 root root          0 Sep  5 18:50 104706425
-rw-rw-rw- 1 root root 1291191296 Sep  5 19:11 3863984572
```
There seems to always only be 1 file that has > 0 size. Can someone explain what's happening here? Why does this post-download step take so long, and can it be safely skipped?

**Comment by chrisd8088** [comment]
Hey, I'm sorry you're having trouble.  In order to help more, we normally ask that you provide the output of `git lfs env`, run from inside the repository's working tree, as well as configuration settings from `git config -l` (with any private information redacted).

As well, I'd suggest running your command with the `GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1` environment variables set to capture a trace log and providing that as well, again with any authorization or other private details removed.
